src/main/resources/i18n/Enums_en.properties

```diff
@@ -142,6 +149,10 @@ public.SpeciesProblemField.ScientificName=Scientific Name
 public.SpeciesProblemType.NameIsSynonym=Name Is Synonym
 public.SpeciesProblemType.NameMisspelled=Name Misspelled
 public.SpeciesProblemType.NameNotFound=Name Not Found
+public.SuccessionalGroup.EarlySecondary=Early secondary
+public.SuccessionalGroup.LateSecondary=Late secondary
+public.SuccessionalGroup.Mature=Mature
+public.SuccessionalGroup.Pioneer=Pioneer
```

Comment:
Once these are translated, we'll probably want to have someone with both domain and language knowledge double-check the translations since the Spanish technical terms for these concepts may be words that don't mean the same things as the everyday meanings of the English words here. (Or maybe the obvious translations are the correct ones; I don't know.)

Reply:
https://terraformation.atlassian.net/browse/SW-5306